packages/website/components/GlobalStyles.tsx

```diff
@@ -48,6 +48,20 @@ export const GlobalStyles = () => {
           margin: 0;
         }
 
+        p,
+        a,
+        li {
+          font-family: ${tokens.fontStackPrimary};
```

Comment:
if we want these styles to be applied only for MDX pages, it's better to apply re-styling in `MdxRenderer` components map.

Right now, these overrides affect how our components look like.